pkg/addons/applier.go

```diff
@@ -52,8 +52,16 @@ type Applier struct {
 	privateCAs      map[string]string
 }
 
+type OutroOptions struct {
+	Ctx              context.Context
```

Comment:
ctx shouldn't be in the options struct from a general code standards department
moving the rest to one potentially makes sense but also makes it dramatically harder to ensure you've updated every caller